rpa-builder/6.4/modules/ROOT/pages/toolbox-variable-handling-json-query.adoc

```diff
@@ -1,48 +1,49 @@
 
-
 = Json Query
 
-The _Json Query_ Action Step enables you to parse Json objects and to
-read individual values / elements from the object. The result can then
-be used in the remainder of the process run as a source variable.
+Use the Json Query Action Step to parse Json objects and read individual values or elements from the object. The result remains available for the rest of the process run as a source variable.
 
-This Action Step can be combined particularly well with the _REST Call_
-Action Step in order to further refine the result that is output there.
+Combine this Action Step with REST Call to further refine the resulting JSON output.
 
 == Properties
 
-*JsonPath Expression* Displays the JsonPath Expression, for example,
-$.access_token.
+* *JsonPath Expression*
++
+The JsonPath expression. For example, `$.access_token`.
 
 == Inbound Variables
 
-*Json Object* From this Json object, the required value is extracted
-using the JsonPath Expression. The result of a REST Call Action Step,
-which you can select from the drop-down list, serves as an object.
+* *Json Object* 
++
+The Json object from which to extract the value using the JsonPath expression. You can select the result of a REST Call Action Step to serve as an object.
 
-*JsonPath Expression* Here, you use the JsonPath Expression to specify
-which value is to be extracted from the object, for example, $.Id., if
-you only require the ID as a result.
+* *JsonPath Expression* 
++
+The JsonPath expression to specify the value to extract from the object. For example, `$.ID`, to extract the value of element `ID` as a result.
 
 == Outbound Variables
 
-*IsSucceeded* (boolean) Shows whether the Action Step has been
-successfully run or not.
+* *IsSucceeded* (boolean) 
++
+Shows if the Action Step executed successfully or not.
 
-*FirstItem* (array) The first element of the array is output as a string
+* *FirstItem* (array) 
++
+Outputs the first element of the array as a string.
 
-*ArrayCount* Indicates the number of elements of the returned array.
+* *ArrayCount* 
++
+Indicates the number of elements of the returned array.
 
 == Wizard
 
-See <<Inbound Variables>>.
-
-Note that an empty result is still a valid result and is not treated as an error.
+Use the Wizard to select the source JSON Object, usually an Inbound Variable containing the result of a REST Call, and to specify the JsonPath expression to extract values from the object. An empty result is a valid result and the process continues without considering it as an error.

```

Comment:
Why all the caps? 
inbound variable that contains the result of a REST call

An empty result is valid and...
Do you need to say that the rest of the process continues without considering it as an error? Being valid implies this.